src/hearings/utils/validators.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@angular/core';
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import * as moment from 'moment';
+import { HearingDateEnum } from '../models/hearings.enum';
+
+@Injectable({ providedIn: 'root' })
+export class ValidatorsService {
```

Comment:
could this  validators.service.ts rename to validators.utils.ts? same as *.spec.ts.